pandas/tests/plotting/test_datetimelike.py

```diff
@@ -1222,6 +1222,14 @@ def test_secondary_y_irregular_ts_xlim(self):
         self.assertEqual(left, ts_irregular.index.min().toordinal())
         self.assertEqual(right, ts_irregular.index.max().toordinal())
 
+    def test_plot_outofbounds_datetime(self):
```

Comment:
In `TestPlotBase` (from which this testcase inherits), all settings are set to the defaul in the setup, and the figures closed in the teardown. 
Or are there still other things that may leak?